src/components/newContractor/NewContractorForm.js

```diff
@@ -0,0 +1,134 @@
+import React, { useContext} from 'react';
+import PropTypes from 'prop-types'
+import { useFormik } from 'formik'
+import * as Yup from 'yup'
+
+import Card from '../layout/card/Card'
+import Input from '../layout/form/Input'
+import Button from '../layout/button/Button'
+import AlertContext from '../../context/alert/alertContext'
+
+import './newContractorForm.css'
+
+const NewContractorForm = (props) => {
+
+    // variables for Alert display - AlertContext
+    const alertCtx = useContext(AlertContext)
+    const { setAlertMessage } = alertCtx
+
+        const formik = useFormik({
+        initialValues: {
+            companyName: '',
+            NIP: '',
+            street: '',
+            post: '',
+            city: '',
+            tel: ''
+        },
+
+        validationSchema: Yup.object({
+            companyName: Yup.string()
+                .max(30, "Maksymalnie 30 znaków")
+                .required('Required'),
+            NIP: Yup.string()
+                .required('Required'),
```

Comment:
this should be 9 digit number

Reply:
you don't have to worry that in input you have string type. Yup will automatically try to cast string to number, eg. `"123"` to `123`

Reply:
stays as it is